packages/feedback/src/core/createMainStyles.ts

```diff
@@ -1,5 +1,5 @@
-import type { FeedbackInternalOptions } from '@sentry/types';
 import { DOCUMENT } from '../constants';
+import type { FeedbackInternalOptions } from '../types';
```

Comment:
typical change for this diff: `from '@sentry/types';` becomes `from '../types';`